app/uk/gov/hmrc/gform/views/group.scala.html

```diff
@@ -29,7 +28,9 @@ <h2 class="heading-small">
         </div>
       </div>
     </div>
-  } else {
+  }
```

Comment:
The problem with deleting that "else" is that non-repeating groups don't get a label and unless I am mistaken, they need a label too

Reply:
would fieldValue.label return a boolean aswell ?

Reply:
@tenochdlc Both will get labels now, with this revised logic. But if it is not repeating and the label is empty, then it will not display it